lib/SmartyPlugins/LMSSmartyPlugins.php

```diff
@@ -329,6 +329,7 @@ public static function fileUploadFunction(array $params, $template)
         }
 
         $form = isset($params['form']) ? $params['form'] : null;
+        $image_resize = isset($params['image_resize']) && empty($params['image_resize']) ? false : true;
```

Comment:
Można tak:
```php
$image_resize = !isset($params['image_resize']) || !empty($params['image_resize']);
```

Reply:
Dzięki. Zapamiętam :-)

Reply:
Zapamiętywać nie warto - nic wielkiego - kwestia optymalizacji.

Reply:
Chodziło mi o to, że zapamiętam, żeby częściej korzystać z takich optymalizacji :-)